packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -511,6 +515,23 @@ export default function useBlockDropZone( {
 		200
 	);
 
+	/**
+	 * Checks if the given element is an insertion point.
+	 *
+	 * @param {EventTarget|null} targetToCheck - The element to check.
+	 * @param {Document}         ownerDocument - The owner document of the element.
+	 * @return {boolean} True if the element is a insertion point, false otherwise.
+	 */
+	function isInsertionPoint( targetToCheck, ownerDocument ) {
```

Comment:
(Very) tiny optimisation idea: this appears to be a pure function. Could it be moved outside of the body of the hook, since it doesn't depend on anything within it?

Reply:
Yes you're right. The new location of the function in the consumer will allow this to happen 👍